testsuite/gnat2goto/tests/range_third/test.opt

```diff
@@ -0,0 +1,2 @@
+ALL XFAIL Doesn't crash anymore but cbmc does not perform range assertion checks 
+
```

Comment:
@jeremycrisp worth keeping in mind.

Reply:
I felt the purpose of the test was to check the loop and therefore felt a positive result was required.  I have added a negative test in which the assertion fails.